x-pack/plugins/security_solution/public/management/components/console/components/console_state/types.ts

```diff
@@ -36,6 +36,9 @@ export interface ConsoleDataState {
 
   dataTestSubj?: string;
 
+  /** The local storage prefix for saving/persisting data associated with the console */
+  storagePrefix?: string;
```

Comment:
why is it called a storage prefix?

Reply:
@parkiino ,

The idea here is that data will only be persisted to local storage if this prefix is defined. This prefix will become part of the localstorage key that is used for the console. Currently, only the input history is saved to local storage, but there could be other data we save in the future (ex. preferences, user command aliases, etc.), so instead of adding a prop to the console (FYI: it was also added to the top-level `<Console/>` component) specific for command input history, I added instead just the prefix and if it is defined, then we persist certain things to localstorage (currently only the input history).